packages/headless/src/features/facets/category-facet-set/category-facet-set-selectors.ts

```diff
@@ -15,7 +15,7 @@ function isCategoryFacetResponse(
 }
 
 export const categoryFacetResponseSelector = (
-  state: CategoryFacetSection & Partial<FacetResponseSection>,
+  state: CategoryFacetSection & Partial<SearchSection>,
```

Comment:
What's the impact of this change exactly? and how is it related to the removal of the product-listing?

Reply:
FacetResponseSection used to contain both SearchSection and OldProductListingSection. 
<img width="678" alt="image" src="https://github.com/user-attachments/assets/f6432eaf-7995-4227-8158-ecc25a6ee977">. this is just typescript stuff anyway, it has no impact.